submissions/tosinKolawole/html-forms/product-page.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset=@"UTF-8">
+    <title>Lydia Stores</title>
+</head>
+<body>
+      <header>
+            <h1><b>LYDIA STORES</b></h1>
+            <div>
+              <ul>
+                 <li><a href="file:///C:/Users/TOSIN/Desktop/Dufuna/index%20product%20pg.html">Product</a></li> 
+                 <li><a href="file:///C:/Users/TOSIN/Desktop/Dufuna/html-forms/html-forms.html">Contact us</a></li><br>
+               </ul>
+              </div>
+          <section>
+           <div>
+            <ul>
+                  <li><a href="https://web.facebook.com/public/Welcome-To-Facebook?_rdc=1&_rdr">Facebook</a></li> 
+                  <li><a href="https://twitter.com/login">Twitter</a></li>
+                  <li><a href="https://www.instagram.com/accounts/login/">Instagram</a></li><br>
+                </ul>
+           </div>  
+            <H3><a href="file:///C:/Users/TOSIN/Desktop/Dufuna/index%20product%20pg.html">INFINIX NOTE 5</a></H3>
```

Comment:
This should be h3 and not H3. Also, this link will not work...similar reason as earlier stated